### PR TITLE
feat(codeblock): add basic code block

### PR DESCRIPTION
**Pull request by oysteinsigholt**
screenshot from 2017-06-30 09-50-47

### CODE REVIEW

**Comment by oysteinsigholt** [comment]
Should .tn-codeblock reside in the tn-code.scss or should it get its own file?

File: components/typography/tn-code-mixins.scss [hunk 1]
```diff
@@ -8,3 +9,7 @@
   background-color: $tn-color--light-gray-background;
   font-family: monospace;
 }
+
+@mixin tn-codeblock() {
```
Comment (tipsy) [inline components/typography/tn-code-mixins.scss]: tn-pre

File: components/typography/tn-code.scss [hunk 2]
```diff
@@ -11,3 +11,17 @@
...
+
+/**
+ * Light gray block with monospaced font used to display code
+ *
+ * @atom Basic code block
+ * @section Typography
+ * @markup
+ *  <pre class="tn-codeblock"><code>Example code</code></pre>
+ */
+.tn-codeblock {
```
Comment (tipsy) [inline components/typography/tn-code.scss]: tn-pre

File: components/typography/tn-code.scss [hunk 3]
```diff
@@ -11,3 +11,17 @@
...
   @include tn-code;
 }
+
+/**
+ * Light gray block with monospaced font used to display code
+ *
+ * @atom Basic code block
+ * @section Typography
+ * @markup
+ *  <pre class="tn-codeblock"><code>Example code</code></pre>
```
Comment (tipsy) [inline components/typography/tn-code.scss]: tn-pre

File: components/typography/tn-code.scss [hunk 4]
```diff
@@ -11,3 +11,17 @@
...
+ */
+.tn-codeblock {
+  @include tn-border-box-everything;
+  @include tn-code;
+  @include tn-codeblock;
```
Comment (tipsy) [inline components/typography/tn-code.scss]: tn-pre

**Comment by tipsy** [comment]
> Should .tn-codeblock reside in the tn-code.scss or should it get its own file?

I think you can keep it in `code.scss`

**Comment by pkkummermo** [comment]
I kinda like them to have separate files to be able to include only one element and not the other.

**Comment by pkkummermo** [comment]
@oysteinsigholt can you fix the conflicts?

**Comment by tipsy** [comment]
The online editor is killing me, will fix locally.

**Pull request merged**